conf/application.conf

```diff
@@ -145,7 +145,7 @@ cache {
   isEnabled = true
 }
 
-mongo.encryption.enabled = true
+mongo.encryption.enabled = false
```

Comment:
Should this still be encrypted?

Reply:
yeah its overriden in base, left it as false locally so we can actually debug mongodb in TAI